test/e2e/specs/delete-prohibited/recordset/histogram-facet.spec.js

```diff
@@ -300,5 +301,23 @@ describe("Viewing Recordset with Faceting,", function() {
                 })(testParams.facets[j], j);
             }
         });
+
+        describe ("going to a page with filters that return no data.", function () {
+            beforeAll(function () {
+                var noDataUri = uri + "/int_col::leq::0&float_col::leq::0&date_col::leq::2000-01-01&timestamp_col::leq::2000-01-01";
+                browser.get(noDataUri);
+                chaisePage.waitForElementInverse(element(by.id("spinner")));
+            });
+
+            it ("no data should be represented to the user.", function () {
+                expect(chaisePage.recordsetPage.getRows().count()).toBe(0);
+            });
+
+            it ("no histogram should be displayed.", function () {
+                for(var i = 0; i < testParams.totalNumFacets; i++) {
+                    expect(chaisePage.recordsetPage.getHistogram(i).isPresent()).not.toBeTruthy("histogram is available for facet index=" + i);
```

Comment:
This is subjective, but why use `.not.toBeTruthy()` when you could use `.toBeFalsy()`?

Note: to me, `toBeFalsy()` looks like it's spelt wrong and not a real function. But we do use it in numerous places in the tests.

Reply:
I forgot about `toBeFalsy`. I will change it.